lib/themes/dosomething/paraneue_dosomething/js/components/Slideshow.js

```diff
@@ -20,9 +20,12 @@ class Slideshow extends React.Component {
   }
 
   render() {
+    const user = this.props.user;
+    const campaign = this.props.campaign;
+
     return (
       <div className="slideshow">
-        {this.props.slides[this.state.activeSlide]}
+        {React.createElement(this.props.slides[this.state.activeSlide], {campaign: campaign, user: user})}
```

Comment:
I honestly made this mistake for a while at first, I didn't even know key was a thing you had to do until I read it in a blog post. I think this is one of those gotchas that isn't explained well